src/main/java/cn/edu/thu/tsfile/timeseries/write/InternalRecordWriter.java

```diff
@@ -123,7 +123,7 @@ protected void checkMemorySize() throws IOException {
 				flushRowGroup(true);
 				recordCountForNextMemCheck = rowGroupSizeThreshold / oneRowMaxSize;
```

Comment:
checkMemorySize method in this class should add a constraint on record count. Flush rowgroup When record count reaches the threshold